app/src/test/java/kantwonskids/donationtrackerg14b/getLocationByNameUnitTest.java

```diff
@@ -0,0 +1,86 @@
+package kantwonskids.donationtrackerg14b;
+
+import kantwonskids.donationtrackerg14b.model.Location;
+import kantwonskids.donationtrackerg14b.model.Model;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.Assert.*;
+
+/**
+ * @author Juliana Petrillo
+ * @version 1.0
+ *
+ * JUnit Test to evaluate functionality of the Model class's
+ * getLocationByName(String name) method.
+ */
+public class getLocationByNameUnitTest {
```

Comment:
IDK if there's a guideline for it, but I think we're supposed to name the test class by the class that it's testing, not the method. The test methods themselves would then be <methodName>SomethingTest().